test/els_diagnostics_SUITE.erl

```diff
@@ -53,7 +54,23 @@ end_per_suite(Config) ->
 -spec init_per_testcase(atom(), config()) -> config().
 init_per_testcase(TestCase, Config) ->
   mock_notifications(),
-  els_test_utils:init_per_testcase(TestCase, Config).
+  Fun = fun () ->
```

Comment:
The project uses 2-spaces indentation, not 4, so we should try to be consistent.
@jfacorro is it possible to enforce this via Elvis?

Reply:
I never used editorconfig, but I'm all for it.